website/src/pages/api/messages/user.ts

```diff
@@ -2,10 +2,25 @@ import { withoutRole } from "src/lib/auth";
 import { createApiClientFromUser } from "src/lib/oasst_client_factory";
 import { getBackendUserCore } from "src/lib/users";
 
+const LIMIT = 10;
+
 const handler = withoutRole("banned", async (req, res, token) => {
   const user = await getBackendUserCore(token.sub);
   const client = createApiClientFromUser(user);
-  const messages = await client.fetch_my_messages(user);
+  const { cursor, direction, include_deleted } = req.query;
+
+  let messages;
+  if (typeof cursor === "string") {
+    messages = await client.fetch_my_messages_cursor(user, {
+      include_deleted: include_deleted === "true",
```

Comment:
I believe we should not expose this until #1884 has been discussed

Reply:
I can remove sure, but note that this is only in the api, and `fetch_message_tree` also has `include_deleted` in the api.  Was trying to be consistent.  Want me to remove anyway?  Thanks